WordPress/wordcamp.org
# Sessions: Gutenberg-ify Session Info & Speakers metaboxes

This is an attempt at switching over to using a gutenberg pattern for the session meta info (date, time, etc). It sets up a gutenberg plugin which slots into the Document Settings sidebar, and creates panels for Session Info & Speakers.

The idea is that we could phase out the metabox & meta saving functionality, and instead expose the meta via API + `register_post_meta` and use the UI patterns of Gutenberg.

### Screenshots

![Screen Shot 2020-01-14 at 4 56 17 PM](https://user-images.githubusercontent.com/541093/72386997-820a8580-36f0-11ea-9239-d7b7aca408b3.png)

### How to test the changes in this Pull Request:

1. Edit a session (either new, or existing)
2. If existing, the session info should correctly populate the new panel
3. If new, it should be empty
4. Edit the session info, save
5. It should save correctly
